src/vivarium/framework/randomness/manager.py

```diff
@@ -33,6 +33,7 @@ class RandomnessManager(Manager):
             "key_columns": [],
             "random_seed": 0,
             "additional_seed": None,
+            "rate_conversion_type": None,
```

Comment:
We want to find a better place to configure this, since this doesn't intrinsically have anything to do with randomness.

Reply:
Decided we will leave this here for now